README.md

```diff
@@ -136,10 +146,22 @@ Workmanager().registerOneOffTask("1", "simpleTask", initialDelay: Duration(secon
 
 > Not all constraints are mapped.
 
+- NetworkType
+  Constrains the type of network required for your work to run. For example, Connected.
```

Comment:
Can you please add a comment for iOS here. iOS background work can not be configured as fine-grained as Android. On iOS you can "requireNetwork" but that's it. It does not allow configuration of unmetered, not_metered etc. All these network variables will be merged into `requiresNetwork`

Reply:
Done